_resources/port1.0/group/cmake-1.1.tcl

```diff
@@ -118,11 +119,16 @@ proc cmake::handle_generator {option action args} {
                 destroot.env-append DESTDIR=${destroot}
             }
             default {
+                if {[llength ${args}] > 1 || [llength ${args}] <= 0} {
```

Comment:
Agreed. Doing pattern matching on a list is also confusing on a conceptual level.